### ISSUE TITLE
Add option to change table of contents to the left side of page

### ISSUE THREAD

**Issue by haekal-alg** [526avijitgupta/gokarna#149]
The current table of content is positioned to the **right** side of the page contrary to the usual place which is to the **left**. 

It would be nice to make this an option so the user could manually set the global position of ToC to the left or to the right.

**Comment by mansoorbarri** [comment]
@yashmehrotra ?

**Comment by yashmehrotra** [comment]
@mansoorbarri Agree with you

This makes it look more like documentation than a blog, if there is demand for it we can put this as an option but does not look very good

**Comment by mansoorbarri** [comment]
should I make a PR then? or not?

**Comment by 526avijitgupta** [comment]
I agree with @yashmehrotra let's hold off the change for now, unless it's requested by more people

**Comment by jamesericdavidson** [comment]
@mansoorbarri's [proposal](https://github.com/gokarna-theme/gokarna-hugo/issues/149#issuecomment-1807733025) could naively fix #234.

If we don't want to add this as an option (hidden behind a config key), should we close as **not planned**? @526avijitgupta @yashmehrotra
